tests/baselines/reference/classFieldSuperNotAccessibleJs.errors.txt

```diff
@@ -0,0 +1,47 @@
+index.js(20,22): error TS2855: Class field 'roots' defined by the parent class is not accessible in the child class via super.
+index.js(23,22): error TS2855: Class field 'foo' defined by the parent class is not accessible in the child class via super.
+index.js(26,22): error TS2855: Class field 'justProp' defined by the parent class is not accessible in the child class via super.
+index.js(29,22): error TS2855: Class field ''literalElementAccess'' defined by the parent class is not accessible in the child class via super.
+
+
+==== index.js (4 errors) ====
+    // https://github.com/microsoft/TypeScript/issues/55884
+    
+    class YaddaBase {
+        constructor() {
+            this.roots = "hi";
+            /** @type number */
+            this.justProp;
+            /** @type string */
+            this['literalElementAccess'];
+            
+            this.b()
+        }
+        accessor b = () => {
+            this.foo = 10
+        }
+    }
+    
+    class DerivedYadda extends YaddaBase {
+        get rootTests() {
+            return super.roots;
+                         ~~~~~
+!!! error TS2855: Class field 'roots' defined by the parent class is not accessible in the child class via super.
+        }
+        get fooTests() {
+            return super.foo;
+                         ~~~
+!!! error TS2855: Class field 'foo' defined by the parent class is not accessible in the child class via super.
+        }
+        get justPropTests() {
+            return super.justProp;
+                         ~~~~~~~~
+!!! error TS2855: Class field 'justProp' defined by the parent class is not accessible in the child class via super.
+        }
+        get literalElementAccessTests() {
+            return super.literalElementAccess;

```

Comment:
Can you add a test for `super["literalElementAccess"]` as well?

Also, can you just add a test for `super.b`? I bet we have no test for `super.` on an `accessor` declaration at all.

Reply:
> Can you add a test for super["literalElementAccess"] as well?

Good catch. It doesn't work correctly today ([TS playground](https://www.typescriptlang.org/play?ts=5.3.0-dev.20230927#code/MYGwhgzhAEAq0G8BQ1oDMCWBTEATaAvNABQCUhAfIgL4DcS1SokMsATNFgB4AuWAdrlaIU6MiNSoIAVwAOWAE4A6TDlxl6k6DPkKA2gCJVeAwF0NoxoyA)) even in TS files. For that reason - I'd say that this is a separate issue. I can look into it as well but I'd prefer to explore it separately.

> Also, can you just add a test for super.b? I bet we have no test for super. on an accessor declaration at all.

Sure thing, just added it.